lib/web_ui/lib/src/engine/canvaskit/fonts.dart

```diff
@@ -11,13 +11,32 @@ import '../util.dart';
 import 'canvaskit_api.dart';
 import 'font_fallbacks.dart';
 
-// This URL was found by using the Google Fonts Developer API to find the URL
-// for Roboto. The API warns that this URL is not stable. In order to update
-// this, list out all of the fonts and find the URL for the regular
-// Roboto font. The API reference is here:
-// https://developers.google.com/fonts/docs/developer_api
-const String _robotoUrl =
-    'https://fonts.gstatic.com/s/roboto/v20/KFOmCnqEu92Fr1Me5WZLCzYlKw.ttf';
+// Runtime parameters.
+//
+/// This URL was found by using the Google Fonts Developer API to find the URL
+/// for Roboto. The API warns that this URL is not stable. In order to update
+/// this, list out all of the fonts and find the URL for the regular
+/// Roboto font. The API reference is here:
+/// https://developers.google.com/fonts/docs/developer_api
+///
+/// The base URL can be overridden using the `ROBOTO_FONT_URL`
+/// environment variable.
+///
+/// When specifying using the environment variable set it in the Flutter tool
```

Comment:
Prefer "The environment variable can be set in the Flutter tool..." rather than "Set [the environment variable] in the Flutter tool". See https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-the-passive-voice-recommend-do-not-require

Reply:
done.